### DISCUSSION TITLE
It's currently unsupported to use "export *" in a client boundary. Please use named exports instead.

### DISCUSSION THREAD

**Discussion by yasoll** [wevm/wagmi#3207] [Q&A] [answered]
I exported the chains from wagmi but it reported an error
Error: It's currently unsupported to use "export *" in a client boundary. Please use named exports instead.

<img width="788" alt="image" src="https://github.com/wagmi-dev/wagmi/assets/127667530/165599c3-c8ad-40db-ad71-bc02b263ceec">

here is my code

wagmi version is v1.4.6

``` ts
import { getDefaultWallets } from '@rainbow-me/rainbowkit';
import { configureChains, createConfig } from 'wagmi';
import {
  mainnet,
  polygon,
  arbitrum
} from 'wagmi/chains';
import { publicProvider } from 'wagmi/providers/public';
export const { chains, publicClient } = configureChains(
  [mainnet, polygon, arbitrum],
  [publicProvider()]
);
const { connectors } = getDefaultWallets({
  appName: "name",
  projectId: "123123123123123",
  chains
});
export const wagmiConfig = createConfig({
  autoConnect: true,
  connectors,
  publicClient
})
```

**Comment by ghost** [comment, marked as answer]
I solved it by adding `'use client'` whenever I import from wagmi.

**Reply by Codensity30** [reply]
Thanks, for answering, but can you please answer where we should import from `wagmi`, in the `layout.tsx` or `page.tsx` cause in `next.js` we don't have a concept of the `App` component?
